manifests/alertmanager.pp

```diff
@@ -0,0 +1,242 @@
+# Class: prometheus::mysqld_exporter
```

Comment:
the name of the binary and package is `alertmanager` not `alert_manager`. This was to keep consistency and allows for easier use of the daemon.pp.